### DISCUSSION TITLE
Fundamentos de C#, Erro ao salvar arquivo em editor de textos

### DISCUSSION THREAD

**Discussion by lcs121** [balta-io/forum#336] [Tira Dúvidas] [answered]
O programa de editor de textos está funcionando normalmente, ate quando chega a hora de salvar o arquivo , pelo que eu vi no video da aula e em outros debates parece ser um erro onde não esta achando a pasta determinada porém eu já mudei diversas vezes e continua o mesmo erro .

![61823e30-e201-4ac8-9bcc-3e3ba0c652c1](https://github.com/balta-io/forum/assets/134514253/296211d6-c00c-4fbb-93e7-1a66065bf155)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @lcs121, como vai?
Tem como informar a mensagem completa? Ficou cortada na imagem.

**Reply by lcs121** [reply]
Coloquei a imagem completa agora professor

**Reply by andrebaltieri** [reply]
Bom dia, @lcs121, como vai?

O erro em questão é que não conseguiu encontrar o caminho ou faltou o nome do arquivo. Veja que ele fala que o caminho "C:\users\lucas" que deu problema.

Em adicional, verifica se você tem permissão para escrever nesta pasta.

**Comment by lcs121** [comment]
Em qui., 25 de mai. de 2023 às 09:17, Andre Baltieri <
***@***.***> escreveu:

> Bom dia @lcs121 <https://github.com/lcs121>, como vai?
> Tem como informar a mensagem completa? Ficou cortada na imagem.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/forum/discussions/336#discussioncomment-6000338>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BACIMTLQBQK6MCN3J267IITXH5EUVANCNFSM6AAAAAAYNNKSVE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
Bom dia professor, tem sim , vou te mandar segue a imagem completa
